Scripts/DCS-BIOS/lib/modules/common_modules/NS430.lua

```diff
@@ -62,7 +68,7 @@ local NS430 = Module:new("NS430", 0x0600, ns430_aircraft)
 function NS430:defineDoubleCommandButton(identifier, device_id, start_command, stop_command, arg_number, category, description)
 	local alloc = self:allocateInt(1, identifier)
 	self:addExportHook(function(dev0)
-		alloc:setValue(dev0:get_argument_value(arg_number))
+		alloc:setValue(math.abs(dev0:get_argument_value(arg_number)))
```

Comment:
Shouldn't we be adjusting the range we're mapping to here instead? Otherwise if one output is -1 and the other is 1 we'll just always send 1 and the exported data won't be correct.

Reply:
There are no ranges here no?

Reply:
Removed ABS()